db/schema.rb

```diff
@@ -76,6 +76,9 @@
   add_index "documents", ["documentable_id", "documentable_type"], name: "index_documents_on_documentable_id_and_documentable_type", using: :btree
   add_index "documents", ["label"], name: "index_documents_on_label", using: :btree
 
+# Could not dump table "events" because of following StandardError
+#   Unknown type 'action_enum' for column 'action'
+
```

Comment:
:warning: a solution is to stop using `db/schema.rb` and start using `db/structure.sql`, let's talk about it

cc/ @sauloperez

Reply:
Another solution is to use the AR built in [enum](http://api.rubyonrails.org/v4.2/classes/ActiveRecord/Enum.html), basically storing an `integer` instead of a `string` (classic Rails approach with validation only at model level) or `ENUM` (DB check level I'm proposing)

Reply:
I think we are safe enough using AR enum :)

Reply:
There is another option using AR `enum` storing `integer` as I said AND adding check inclusion instead of PG `enum`. Will propose this option as it seems a good tradeoff to me